appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/CommitDelayingResponse.java

```diff
@@ -77,9 +77,6 @@
    */
   public CommitDelayingResponse(HttpServletResponse response) throws IOException {
     super(response);
-    // Set the buffer size explicitly, as this ensure both that jetty's aggregate
-    // buffer is the correct size and that the commit threshold is that size.
-    response.setBufferSize(CommitDelayingOutputStream.MAX_RESPONSE_SIZE_BYTES);
```

Comment:
@gregw What is the default commit threshold in Jetty without setting the response buffer size? If it is very small, I feel like we should've been hearing about the truncation problem all the time.